bundles/org.openhab.binding.emotiva/README.md

```diff
@@ -29,18 +37,33 @@ There are more parameters which all have defaults set.
 | activateFrontBar       | Activates Front Bar channels                                  | false   |
 | activateOSDMenu        | Activates OSD menu channels                                   | false   |
 | activateZone2          | Activates Zone 2 channels                                     | false   |
-| protocolVersion        | Emotiva Network Protocol version, e.g. 3.0                    | 2.0     |
+| protocolVersion        | Emotiva Network Protocol version, 2.0 or 3.0 supported        | 3.0     |
 | keepAlive              | Time between notification update from device, in milliseconds | 7500    |
 | retryConnectInMinutes  | Time between connection retry, in minutes                     | 2       |
 
 ### Dynamic channels and extended functionality
 
-Emotiva processors have limited processing power, so if the binding subscribes to all channels simultaneously the 
-device might grind to a halt after a while, requiring a manual reboot of the device. The binding is designed to 
-dynamically enable and disabled channels based on the selected source. Furthermore, the configuration values 
-**activateFrontBar**, **activateOSDMenu** and **activateZone2** controls extra functionality this is default off to 
+Emotiva processors have limited processing power, so if the binding subscribes to all channels simultaneously the
+device might grind to a halt after a while, requiring a manual reboot of the device. The binding is designed to
+dynamically enable and disabled channels based on the selected source. Furthermore, the configuration values
+**activateFrontBar**, **activateOSDMenu** and **activateZone2** controls extra functionality this is default off to
 reduce the overall load on the device.
 
+### Protocol Version
+
+Protocol version is usually reported by the device to the binding during discovery, but if a device is added manually, 
+version 3.0 is set by default. Most modern Emotiva devices supports Emotiva Network Protocol version 3.0, but some
+devices with older firmware versions might require version 2.0.
+
+### Keep Alive
+
+A job to monitor the availability of the device is started whenever a successful connection is made. This monitors the
+regular a notification message received via the Control Protocol, and whenever a keep alive message is not received, the
+device is set OFFLINE. This might be an indication the device either has been disconnected or has been turned off
+manually. The binding will retry connecting to the device every 2 minutes, so if a device is turned back on it will
+automatically be discovered by the binding. If the devices goes away within minutes after an initial connection,
+this might be an indication that the **notifyPort** has not been opened in the firewall.
```

Comment:
```suggestion
A job to monitor the availability of the device is started whenever a successful connection is made.
This monitors the regular a notification message received via the Control Protocol, and whenever a keep alive message is not received, the device is set OFFLINE.
This might be an indication the device either has been disconnected or has been turned off manually.
The binding will retry connecting to the device every 2 minutes, so if a device is turned back on it will automatically be discovered by the binding.
If the devices goes away within minutes after an initial connection, this might be an indication that the **notifyPort** has not been opened in the firewall.
```